libs/uniqueness/jpa-backing-store-datamodel/src/main/kotlin/net/corda/uniqueness/backingstore/jpa/datamodel/JPABackingStoreEntities.kt

```diff
@@ -212,11 +211,14 @@ data class UniquenessRejectedTransactionEntity(
     @Column(name = "tx_id", length = TRANSACTION_ID_LENGTH, nullable = false)
     val txId: ByteArray,
 
-    // TODO Removing @Lob annotation from here is a temporary solution, should be investigated as part of NAAS-458
-    // @Lob
     @Column(name = "error_details", nullable = false)
     val errorDetails: ByteArray
 ) {
+    init {
+        if (errorDetails.size > 1024) {
+            throw IllegalArgumentException("The maximum size of an Error detail is 1024")
```

Comment:
```suggestion
            throw IllegalArgumentException("The maximum size of the error_details field is 1024")
```

A bit more specific, so the reader is left in no doubt as to what the problem is.